app/src/main/java/com/example/makeitso/screens/splash/SplashViewModel.kt

```diff
@@ -28,18 +32,31 @@ import javax.inject.Inject
 
 @HiltViewModel
 class SplashViewModel @Inject constructor(
+    configurationService: ConfigurationService,
     private val accountService: AccountService,
     private val logService: LogService
 ) : MakeItSoViewModel(logService) {
+    var showError = mutableStateOf(false)
+        private set
```

Comment:
you are not actually reassigning this var anywhere, so consider making it a val
```suggestion
    val showError = mutableStateOf(false)
```